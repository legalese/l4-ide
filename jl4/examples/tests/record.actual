MkProgram
    [ MkSection 1 "Type declarations"
        [ Declare
            ( MkDeclare "Folder"
                ( Record
                    [ MkTypedName "name"
                        ( NamedType "string" )
                    , MkTypedName "colour"
                        ( Enum
                            [ "red"
                            , "green"
                            , "blue"
                            ]
                        )
                    ]
                )
            )
        ]
    ]
