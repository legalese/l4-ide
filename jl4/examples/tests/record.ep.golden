ยง `Type declarations`

DECLARE Folder
HAS name IS A string
    colour IS ONE OF red, green, blue
